src/Components/EventListPage.js

```diff
@@ -44,20 +45,32 @@ class EventListPage extends Component {
     constructor(){
         super();
         this.state = {
-            location: "Seattle",
+            location: locations.LA.canonical,
             sort: true,
             events: []    
         }
         this.handleChangeRegion = this.handleChangeRegion.bind(this);
         this.setNewEvents = this.setNewEvents.bind(this);
+        this.convertPrettyToCanon = this.convertPrettyToCanon.bind(this);
     }
 
     componentDidMount() {
         this.setNewEvents();
     }
 
     handleChangeRegion(e) {
-        this.setState({ location: e.target.textContent }, this.setNewEvents);
+        this.setState({ 
+            location: this.convertPrettyToCanon(e.target.children[1].innerText)
```

Comment:
This lets us use the canonical form in the HTTP request but still display it as the full form in the element